documentation/modules/ova-prerequisites.adoc

```diff
@@ -19,17 +19,24 @@ Migration of OVA files that were not created by VMware vSphere but are compatibl
 
 ** In one or more compressed Open Virtualization Format (OVF) packages that hold all the VM information.
 +
-The filenames of the compressed packages *must* have `.ova` extensions. Several packages can be stored either in the same folder or in individual folders:
+The filenames of the compressed packages *must* have `.ova` extensions. Several packages can be stored either in the same folder or in individual folders.
 +
-`<nfs_shared_directory>/folder1/vm1.ova` +
-`<nfs_shared_directory>/folder1/vm2.ova` (`vm1.ova` and `vm2.ova` in `folder1`)
+When this structure is used, {project-short} scans the root folder and the first-level subfolders for compressed packages.
 +
-_or_
-+
-`<nfs_shared_directory>/folder1/vm1.ova` +
-`<nfs_shared_directory>/folder2/vm2.ova` (`vm1.ova` in `folder1`, `vm2.ova` in `folder2`)
+For example, if the NFS share is, `/nfs`, then: +
+The folder `/nfs/vm.ova` is scanned. +
+The folder `/nfs/subfolder1` is scanned. +
+But, `/nfs/subfolder1/subfolder2` is not scanned.
 
-** In extracted OVF packages, each package being an individual folder:
+** In extracted OVF packages.
++
+The XML files must be in a separate folder for each VM.
```

Comment:
@bkhizgiy I agree that it would make more sense to place the files this way but your implementation is supposed to work even when placing few extracted OVAs in the same folder, no? I'm looking at [this code](https://github.com/kubev2v/forklift/blob/release-v2.5.1/cmd/ova-provider-server/ova-provider-server.go#L300-L321) - why wouldn't it work if we have a folder with several OVF files and different VMDK files? I wouldn't over-complicate the text here if this structure *can* work

Reply:
the issue is not with the scanning which working fine but with the v2v pod, once a folder has more then one .ovf file the conversion failing with error saying it detected more then one ,ovf file in the path.

Reply:
ah ok, yeah, that makes sense as we don't specify the OVF file but the folder that contains the OVF

> Maybe something like `The XML files must be in a separate folder for each VM. If placed directly under the root path, only one extracted VM can exist for the migration to work.`

so back to this part then, the last part seems like a concrete case of the first part, why is it particularly important? I see it's also mentioned below now ("However, there can be only one `.ovf` file in a folder, otherwise the migration will fail.")

Reply:
@RichardHoch @bkhizgiy if you don't feel attached to this, I'd suggest to drop this line - I think `However, there can be only one `.ovf` file in a folder, otherwise the migration will fail.` is enough

Reply:
@RichardHoch @ahadas yeha agreed, i think is enough, and also more clear that it should be this way for all folders and not only the root one.